### PR TITLE
INF-1135 Fix bug dealing with manually triggered incidents

### PR DESCRIPTION
**Pull request by nerdyness**
Actual code fix:
Instead of relying on the log message of type == trigger which doesn't exist in manually triggered incidents, simply fetch the first available time stamp which will be when the incident started.

And a whole lot of Rubocop

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,22 +1,10 @@
...
-
-# Enforce spaces within Braces
-Style/SpaceInsideHashLiteralBraces:
-  EnforcedStyle: space
-
-# Numbers without underscores, please.
+# Numbers with underscores, please.
 Style/NumericLiterals:
-  Enabled: false
+  Enabled: true
```
Comment (mfsgr) [inline .rubocop.yml]: why not just remove the exception altogether?

File: lib/influx.rb [hunk 2]
```diff
@@ -331,17 +327,14 @@ def save_categories(opts)
...
       begin
         entries = @influxdb.query "select * from #{timeseries} where time > #{oldest}s and time < #{newest}s"
         entries = entries.empty? ? [] : entries[timeseries]
       rescue InfluxDB::Error => e
-        if e.message.match(/^Couldn't find series/)
-          entries = []
-        else
-          raise
-        end
+        raise unless e.message =~ /^Couldn't find series/
```
Comment (mfsgr) [inline lib/influx.rb]: :+1:

**Review by ghost — APPROVED** [review]
Looks pretty solid. Mostly rubocop touch ups from what I can see. Only comment is whether or not we can start culling rubocop exceptions. Not a showstopper tho

LGTM :+1:

**Review by nathan333 — APPROVED** [review]
looks pretty straight forward to me

**Pull request merged**